frontend/scss/pages/_home_page.scss

```diff
@@ -199,6 +199,48 @@
   margin-top: 50px;
 }
 
+.products-component-container {
+  margin: 0 auto;
+}
+
+.products-component {
+  margin: 20px 5px 20px 5px;
+  padding: 5px 5px 5px 5px;
+
+  display: inline-block;
+
+  //background-color: #325774;
+  //below is site color, i prefer the above though
+  background-color: #363f46;
+
+  border: 1px solid white;
+}
+
+.products-component-name {
```

Comment:
i like the naming convention here, it's actually called 'BEM', I use a loose version of it, but generally the idea is:

component name
component name + component section name

which is what you have, but the way BEM does it is it uses 2 underscores to distinguish the two:

```
.product-summary {
  ...
}

.product-summary__name {
  ...
}
```

it's also good to avoid nesting css selectors so this is a good approach you have